hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/sigs/order/SigRequirements.java

```diff
@@ -1371,13 +1371,19 @@ && isAlias(party)
     private boolean receivesFungibleValue(
             final AccountID target, final CryptoTransferTransactionBody op) {
         for (final var adjust : op.getTransfers().getAccountAmountsList()) {
-            if (adjust.getAmount() > 0 && adjust.getAccountID().equals(target)) {
+            final var unaliasedAccountNum =
+                    sigMetaLookup.unaliasedAccount(adjust.getAccountID(), null);
+            final var unaliasedTargetNum = sigMetaLookup.unaliasedAccount(target, null);
+            if (adjust.getAmount() > 0 && unaliasedAccountNum.equals(unaliasedTargetNum)) {
                 return true;
             }
         }
         for (final var transfers : op.getTokenTransfersList()) {
             for (final var adjust : transfers.getTransfersList()) {
-                if (adjust.getAmount() > 0 && adjust.getAccountID().equals(target)) {
+                final var unaliasedAccountNum =
+                        sigMetaLookup.unaliasedAccount(adjust.getAccountID(), null);
+                final var unaliasedTargetNum = sigMetaLookup.unaliasedAccount(target, null);
+                if (adjust.getAmount() > 0 && unaliasedAccountNum.equals(unaliasedTargetNum)) {
```

Comment:
It'd be nice not to duplicate the logic here, but not a huge deal

Reply:
Agreed. Modified that